### PR TITLE
Build Range Circle

### PR DESCRIPTION
**Pull request by Voz-Duh**
image
image
image
https://github.com/Anuken/Mindustry-Suggestions/issues/1596

### CODE REVIEW

**Comment by sk7725** [comment]
maybe use Lines.spikes() for a dotted line?
Also 
```
Draw.alpha(Mathf.absin(Time.globalTime, 6f, 0.5f) + 0.5f);
```

**Comment by Voz-Duh** [comment]
Thx for your help

File: core/src/mindustry/graphics/OverlayRenderer.java [hunk 1]
```diff
@@ -107,15 +107,20 @@ public void drawTop(){
...
                     Lines.circle(core.x, core.y - 2, state.rules.enemyCoreBuildRadius);
-                    Draw.color(Pal.accent, core.team.color, 0.5f + Mathf.absin(Time.time, 10f, 0.5f));
+                    Draw.color(Pal.accent, core.team.color, adsin);
                     Lines.circle(core.x, core.y, state.rules.enemyCoreBuildRadius);
                 }
             });
+
+            Draw.color(Pal.accent);
+            Draw.alpha(adsin);
+            Lines.circle(player.x, player.y, 208.2f);
```
Comment (Quezler) [inline core/src/mindustry/graphics/OverlayRenderer.java:123]: what is `208.2f`? is that the build radius for (all?) units? why is it hardcoded in here?

Comment (Voz-Duh) [reply core/src/mindustry/graphics/OverlayRenderer.java:123]: I asked Anuke in what code moment this saved
But Anuken don't told me

Comment (Quezler) [reply core/src/mindustry/graphics/OverlayRenderer.java:123]: https://github.com/Anuken/Mindustry/blob/5654e3120dfa0398e1199a1748c38b6e6b63208e/core/src/mindustry/Vars.java#L86

Comment (Voz-Duh) [reply core/src/mindustry/graphics/OverlayRenderer.java]: Thx

**Comment by Anuken** [comment]
This seems like bloat. The circle style doesn't fit any other 'ranged' object, and it doesn't look particularly useful.

**Comment by joshuaptfan** [comment]
Yeah, players are not going to care about "am I in range" when placing blocks most of the time. If they are out of range, they'll just move until they're not.

**Pull request closed without merging**